### DISCUSSION TITLE
Add config options

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#22] [Ideas]
- Link Paths: Shortest past or relative path for markdown links and wikilinks; Related to #2 and #18

**Comment by sQVe** [comment]
As we've discusses previously: it would be great if one could configure completion for headers or not.

**Comment by Feel-ix-343** [comment]
@sQVe based on your experience, do you think the config file should be part of the LSP settings or in a toml file like marksman (or both, which would just take longer to implement). 

I personally like a config file. The idea for this LS is to be an advanced, Obsidian-inspired PKM system for *any* text editor. Having a config file would allow you to use the same config for multiple text-editors.

**Reply by sQVe** [reply]
I don't mind either, to be honest.

**Comment by Feel-ix-343** [comment]
I think it would be quite neat to import your Obsidian config.

- daily note format
- completion types
- ...

**Reply by sQVe** [reply]
I would prefer a configuration that is outside of Obsidian. 

The reasoning I have is:
- One might use this plugin but not Obsidian
- The configuration option will be locked with Obsidian, which provides minimal configuration for certain areas.

**Reply by Feel-ix-343** [reply]
Yes I should have said that the defaults would be imported from Obsidian, but all options would be configurable in a file as well.

**Comment by Feel-ix-343** [comment]
PR here: #36

**Reply by Feel-ix-343** [reply]
Merged w/ documentation
